doc/building.rst

```diff
@@ -224,7 +224,31 @@ Compared to previous versions, building the desktop sync client has become easie
 earlier versions, CSync, which is the sync engine library of the client, is now
 part of the client source repository and not a separate module.
 
-To build the most up-to-date version of the client:
+To build the most up-to-date version of the client, requires four mandatory steps (and one optional one).
+But, before you can run those, make sure you have all the dependencies installed. 
+These are:
+
+* build-essential
+* bzip2
+* cmake
+* doxygen
+* git
+* gzip
+* lacheck
+* libdbusmenu-qt5-dev 
```

Comment:
It seemed to be required when I was working through the instructions. Perhaps I bundled it in with another package, so it seemed like it was needed, when it wasn't.